.github/workflows/lts-branch-create-pull-request.yaml

```diff
@@ -31,8 +31,8 @@ jobs:
           DESTINATION_BRANCH=$(echo $GITHUB_BRANCH_NAME | cut -f $DEST_BRANCH_FIELD_NUMBER -d $DELIMITER )
           TAG_NAME=$(echo $GITHUB_BRANCH_NAME | cut -f $TAG_NAME_FIELD_NUMBER -d $DELIMITER )
 
-          echo "::set-output name=destination_branch::$(echo $DESTINATION_BRANCH )"
-          echo "::set-output name=tag_name::$(echo $TAG_NAME )"
+          echo "destination_branch=$(echo $DESTINATION_BRANCH )" >> $GITHUB_OUTPUT
+          echo "tag_name=$(echo $TAG_NAME )" >> $GITHUB_OUTPUT
```

Comment:
why was this syntax chosen over the previous?

Reply:
Please include this in the description
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/